.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [["es2015", {"loose": true}]],
+  "plugins": [
```

Comment:
can we change these two plugins to the es3 preset instead like we've done in videojs and contrib-hls now?